tests/simpleKVBC/TesterReplica/internalCommandsHandler.cpp

```diff
@@ -274,6 +271,22 @@ void InternalCommandsHandler::addBlock(VersionedUpdates &verUpdates, BlockMerkle
   const auto newBlockId = m_blockAdder->add(std::move(updates));
   ConcordAssert(newBlockId == currBlock + 1);
 }
+
+bool InternalCommandsHandler::checkKeyInBlockAccumulatedRequests(
```

Comment:
Please rename to something like "hasConflicts" as it returns a boolean value

Reply:
Please find updated code with changes